traffic_portal/app/src/common/modules/table/profilesParamsCompare/TableProfilesParamsCompareController.js

```diff
@@ -0,0 +1,213 @@
+				then(function(result) {
+					this['profile' + profNum + 'Usage'] = result.length + ' delivery services';
+				});
+		} else { // otherwise the profile is used by servers so we'll fetch the server count...
+			serverService.getServers({ profileId: profile.id }).
+				then(function(result) {
+					this['profile' + profNum + 'Usage'] = result.length + ' servers';
+				});
+		}
+	};
+
+	let confirmUpdateProfiles = function() {
+		// ok, this method is fun :)
+		let params = {
+			title: 'Modify ' + profile1.name + ' parameters',
+			message: 'The ' + profile1.name + ' profile is used by ' + this.profile1Usage + '. Are you sure you want to modify the parameters?'
+		};
+		let modalInstance = $uibModal.open({
+			templateUrl: 'common/modules/dialog/confirm/dialog.confirm.tpl.html',
+			controller: 'DialogConfirmController',
+			size: 'md',
+			resolve: {
+				params: function () {
+					return params;
+				}
+			}
+		});
+		modalInstance.result
+			.then(
+				function() { updateProfile1 = true; },
+				function() { updateProfile1 = false; }
+			)
+			.finally(
+				function() {
+					let params = {
+						title: 'Modify ' + profile2.name + ' parameters',
+						message: 'The ' + profile2.name + ' profile is used by ' + this.profile2Usage + '. Are you sure you want to modify the parameters?'
+					};
+					let modalInstance = $uibModal.open({
+						templateUrl: 'common/modules/dialog/confirm/dialog.confirm.tpl.html',
+						controller: 'DialogConfirmController',
+						size: 'md',
+						resolve: {
+							params: function () {
+								return params;
+							}
+						}
+					});
+					modalInstance.result
+						.then(
+							function() { updateProfile2 = true; },
+							function() { updateProfile2 = false; }
+						)
+						.finally(
+							function() {
+								let promises = [];
+
+								if (updateProfile1) {
+									promises.push(profileParameterService.linkProfileParameters(profile1, _.pluck($scope.selectedProfile1Params, 'id')));
+								}
+
+								if (updateProfile2) {
+									promises.push(profileParameterService.linkProfileParameters(profile2, _.pluck($scope.selectedProfile2Params, 'id')));
+								}
+
+								if (promises.length > 0) {
+									$q.all(promises)
+										.then(
+											function(result) {
+												let messages = [];
+												for (let i = 0; i < result.length; i++) {
+													messages = _.union(messages, result[i].data.alerts);
+												};
+												messageModel.setMessages(messages, false);
+											})
+										.finally(
+											function() {
+												$scope.refresh();
+											}
+										);
+								}
+							}
+						);
+				}
+			);
+	};
+
+	$scope.dirty = false;
+
+	$scope.profile1 = profile1;
+	$scope.profile2 = profile2;
+	$scope.profilesParams = profilesParams;
+
+	$scope.selectedProfile1Params = [];
+	$scope.selectedProfile2Params = [];
+
+	$scope.selectedParams = _.map(profilesParams, function(param) {
```

Comment:
instead of using `_.map` you can just directly use the [`Array.prototype.map` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map).